openvpn-install.sh

```diff
@@ -849,9 +849,12 @@ ncp-ciphers $CIPHER
 tls-server
 tls-version-min 1.2
 tls-cipher $CC_CIPHER
+client-config-dir /etc/openvpn/ccd
 status /var/log/openvpn/status.log
 verb 3" >> /etc/openvpn/server.conf
 
+	# Create ccd dir
+	mkdir /etc/openvpn/ccd
```

Comment:
Supress error message for existing directory
```suggestion
	mkdir -p /etc/openvpn/ccd
```